client/me/security-2fa-disable/index.jsx

```diff
@@ -151,18 +151,18 @@ class Security2faDisable extends Component {
 					</p>
 					<p>
 						{ translate(
-							'This will also disable your Application Passwords, ' +
+							'This will also disable your application passwords, ' +
 								'though you can access them again if you ever re-enable ' +
-								'Two-Step Authentication. If you decide to re-enable ' +
-								"Two-Step Authentication, keep in mind you'll need to " +
+								'two-step authentication. If you decide to re-enable ' +
+								"two-step authentication, keep in mind you'll need to " +
 								'generate new backup codes.'
 						) }
 					</p>
 					<p>
 						{ translate(
-							'To verify that you wish to disable Two-Step Authentication, ' +
+							'To verify that you wish to disable two-step authentication, ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'To verify that you wish to disable Two-Step Authentication, enter the verification code from your device or a backup code, and click "Disable Two-Step."' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)